Languages/french.lang

```diff
@@ -72,25 +72,25 @@ msgid "1 hour"
 msgstr "1 heure"
 
 msgid "10 min"
-msgstr "10 min."
+msgstr ""
```

Comment:
This should be
```suggestion
msgstr "10 min"
```
even if it’s the same string, so that various tools that report po file translations don’t show that this one is missing 🙂